interface/main/left_nav.php

```diff
@@ -1358,6 +1358,7 @@ function selpopup(selobj) {
       </li>
       <li><a class="collapsed_lv2"><span><?php xl('Visits','e') ?></span></a>
         <ul>
+          <?php  genMiscLink('RTop','rep','0',xl('Daily Summary Report'),'reports/daily_summary_report.php'); ?>
```

Comment:
Yes, It would be good to display 'Daily Summary' in the left navigation menu. We have made the changes in the left_nav.php file